tests/engine/adhoc_test.go

```diff
@@ -170,3 +172,28 @@ func recursiveEntry(t *testing.T, newEngine func() wasm.Engine) {
 	_, _, err = store.CallFunction("test", "main", uint64(1))
 	require.NoError(t, err)
 }
+
+func importedAndExportedFunc(t *testing.T, newEngine func() wasm.Engine) {
+	buf, err := os.ReadFile("testdata/imported_and_exported_func.wat")
```

Comment:
maybe just backtrack this as for small text inlining > go:embed > os.ReadFile

Reply:
inlined!